osu.Game/Overlays/Dashboard/DashboardOverlayHeader.cs

```diff
@@ -13,18 +16,37 @@ private class DashboardTitle : OverlayTitle
         {
             public DashboardTitle()
             {
-                Title = "dashboard";
+                Title = HomeStrings.UserTitle;
                 Description = "view your friends and other information";
                 IconTexture = "Icons/Hexacons/social";
             }
         }
     }
 
+    [LocalisableEnum(typeof(DashboardOverlayTabsEnumLocalisationMapper))]
     public enum DashboardOverlayTabs
     {
         Friends,
 
         [Description("Currently Playing")]
         CurrentlyPlaying
     }
+
+    public class DashboardOverlayTabsEnumLocalisationMapper : EnumLocalisationMapper<DashboardOverlayTabs>
+    {
+        public override LocalisableString Map(DashboardOverlayTabs value)
+        {
+            switch (value)
+            {
+                case DashboardOverlayTabs.Friends:
+                    return FriendsStrings.TitleCompact;                  
+
+                case DashboardOverlayTabs.CurrentlyPlaying:
+                    return @"Currently Playing";
```

Comment:
As mentionned in the opening post, I left this string as is, since it is from a lazer-specific component. I'll get to localise that one later, same for the localisation of the dashboard overlay description.